packages/gatsby/src/redux/machines/page-component.ts

```diff
@@ -1,17 +1,24 @@
-const {
-  Machine,
-  actions: { assign },
-} = require(`xstate`)
+import { Machine as machine, assign, AnyEventObject } from "xstate"
```

Comment:
any reason to rename `Machine` to lowercase?

Reply:
eslint new-cap rule does not allow functions to be uppercase.